package-lock.json

```diff
@@ -3894,7 +3894,7 @@
     "fastparse": {
       "version": "1.1.2",
       "resolved": "https://registry.npmjs.org/fastparse/-/fastparse-1.1.2.tgz",
-      "integrity": "sha512-483XLLxTVIwWK3QTrMGRqUfUpoOs/0hbQrl2oz4J0pAcm3A3bu84wxTFqGqkJzewCLdME38xJLJAxBABfQT8sQ==",
+      "integrity": "sha1-kXKMWllC7O2FMSg8eUQe5BIsNak=",
```

Comment:
@apb7 PTAL -- this should not be committed right? Please work with @YashJipkate to debug why this happened.

Note also the permissions changes to pre_commit_hook.py in this PR.

Reply:
This might be because I started this branch before merging with develop which had @apb7's check.